Sources/Operators.swift

```diff
@@ -0,0 +1,181 @@
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import CoreGraphics
+import Foundation
+import UIKit
+
+
+// MARK: – ViewPosition Operators
+
+
+infix operator --> : AssignmentPrecedence
+infix operator <|> : AdditionPrecedence
+
+/// Aligns lhs to rhs.
+/// - parameter lhs: The ViewPosition to align.
+/// - parameter rhs: The position to which lhs is aligned.
+public func -->(lhs: ViewPosition, rhs: ViewPosition) {
+    lhs.align(to: rhs)
+}
+
+/// Aligns lhs to the rhs's position on the lhs's superview.
+/// - parameter lhs: The ViewPosition to align.
+/// - parameter rhs: The position on the lhs's superview to which lhs is aligned.
+public func -->(lhs: ViewPosition, rhs: ViewPosition.Anchor) {
+    lhs.align(toSuperviewAnchor: rhs)
+}
+
+/// Aligns lhs to the rhs's position on the lhs's superview.
+/// - parameter lhs: The ViewPosition to align.
+/// - parameter rhs: The position on the lhs's superview to which lhs is aligned.
+public func -->(lhs: ViewPosition, rhs: ViewPosition.OffsetAnchor) {
+    lhs.align(toSuperviewAnchor: rhs.anchor, xOffset: rhs.offset.horizontal, yOffset: rhs.offset.vertical)
+}
+
+public func +(lhs: ViewPosition, rhs: UIOffset) -> ViewPosition {
+    return ViewPosition(view: lhs.view, position: CGPoint(x: lhs.anchor.x + rhs.horizontal, y: lhs.anchor.y + rhs.vertical))
+}
+
+public func -(lhs: ViewPosition, rhs: UIOffset) -> ViewPosition {
+    return ViewPosition(view: lhs.view, position: CGPoint(x: lhs.anchor.x - rhs.horizontal, y: lhs.anchor.y - rhs.vertical))
+}
+
+public func +(lhs: ViewPosition.Anchor, rhs: UIOffset) -> ViewPosition.OffsetAnchor {
+    return ViewPosition.OffsetAnchor(offset: rhs, anchor: lhs)
+}
+
+public func -(lhs: ViewPosition.Anchor, rhs: UIOffset) -> ViewPosition.OffsetAnchor {
+    return ViewPosition.OffsetAnchor(offset: rhs, anchor: lhs)
+}
+
+public func +(lhs: ViewPosition.OffsetAnchor, rhs: UIOffset) -> ViewPosition.OffsetAnchor {
+    return ViewPosition.OffsetAnchor(offset: lhs.offset + rhs, anchor: lhs.anchor)
+}
+
+public func -(lhs: ViewPosition.OffsetAnchor, rhs: UIOffset) -> ViewPosition.OffsetAnchor {
+    return ViewPosition.OffsetAnchor(offset: lhs.offset - rhs, anchor: lhs.anchor)
+}
+
+
+// MARK: - DistributionItem Operators
+
+
+public func <|>(lhs: DistributionItem, rhs: DistributionItem) -> [DistributionItem] {
+    return [lhs, rhs]
+}
+
+public func <|>(lhs: [DistributionItem], rhs: DistributionItem) -> [DistributionItem] {
+    var bothSides = lhs
+    bothSides.append(rhs)
+    return bothSides
+}
+
+public func <|>(lhs: [DistributionItem], rhs: UIView) -> [DistributionItem] {
+    return lhs <|> .view(rhs)
+}
+
+public func <|>(lhs: UIView, rhs: DistributionItem) -> [DistributionItem] {
+    return .view(lhs) <|> rhs
+}
+
+public func <|>(lhs: DistributionItem, rhs: UIView) -> [DistributionItem] {
+    return lhs <|> .view(rhs)
+}
+
```

Comment:
Should `<|>` work with `CGFloatConvertible`s also, and just add in the `.fixed()` around it?  Simplifies the expression and I don't think it would make it confusing.  

@dfed @kathryneh